src/Input/Silk.NET.Input.Common/Enums/DeadzoneMethod.cs

```diff
@@ -5,11 +5,36 @@
 
 namespace Silk.NET.Input.Common
 {
+    /// <summary>
+    /// Available methods to control the deadzone of a control stick.
+    /// </summary>
     public enum DeadzoneMethod
     {
-        // y = x except where |x| is between 0 and d (the deadzone value)
+        /// <summary>
+        /// The traditional deadzone method.
```

Comment:
Hmmm a part of me thinks this should be documented a bit further
```suggestion
        /// The traditional deadzone method, where the reported value is directly proportional with the actual value until the actual value is within the deadzone - the reported value will be 0 in this case.
```